content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -123,9 +123,69 @@ TODO.
 
 ## Camera-driven UI
 
-<div class="release-feature-authors">authors: @TODO</div>
-
-TODO.
+<div class="release-feature-authors">authors: @bardt, @oceantume</div>
+
+Historically, Bevy's UI elements have been scaled and positioned in the context of the primary window, regardless of the camera settings. This approach made some UI experiences — like split-screen multiplayer — difficult to implement, and others — such as having UI in multiple windows — impossible.
+
+We are now introducing a more flexible way of rendering the user interface: Camera-driven UI. Each camera can now have its own UI root, rendering according to its viewport, scale factor, and target — be it a secondary window, or even a texture.
```

Comment:
Non-blocking, but the use of emdashes here feels overabundant, distracting to read, and perhaps unnecessary imo.

```suggestion
Historically, Bevy's UI elements have been scaled and positioned in the context of the primary window, regardless of the camera settings. This approach made some UI experiences like split-screen multiplayer difficult to implement, and others such as having UI in multiple windows impossible.

We are now introducing a more flexible way of rendering the user interface: Camera-driven UI. Each camera can now have its own UI root, rendering according to its viewport, scale factor, and a target which can be a secondary window or even a texture.
```

Reply:
That's a great suggestion, thank you!